cmd/infracost/breakdown_test.go

```diff
@@ -261,6 +262,17 @@ func TestBreakdownTerragruntHCLDepsOutput(t *testing.T) {
 	GoldenFileCommandTest(t, testutil.CalcGoldenFileTestdataDirName(), []string{"breakdown", "--path", path.Join("./testdata", testutil.CalcGoldenFileTestdataDirName())}, nil)
 }
 
+func TestBreakdownTerragruntGetEnv(t *testing.T) {
+	os.Setenv("CUSTOM_OS_VAR", "test")
```

Comment:
You can also set these as test options in the last param to `GoldenFileCommandTest`:
```
&GoldenFileOptions{
    Env: map[string]string{
        "CUSTOM_OS_VAR": "test",
    },
}
		```